.env

```diff
@@ -0,0 +1,12 @@
+SS_DATABASE_CLASS="MySQLDatabase"
```

Comment:
Shouldn't be adding a .env file directly in root of a module

We dynamically create .env file in ci - https://github.com/silverstripe/gha-ci/blob/1/.github/workflows/ci.yml#L657 - may need to make an update in gha-ci to get this to work the way you want things to?

Reply:
Oops, that was just a `git add .` and not double checking if I was adding extra gunk. It's a directly copy of what CI adds already from when I was trying to figure out what was going wrong in CI.

Reply:
Removed